mmv1/third_party/terraform/services/bigquery/test-fixtures/simple/metadata/00000000000000000000.json

```diff
@@ -0,0 +1,6 @@
+{"commitInfo":{"timestamp":1706278148531,"operation":"WRITE","operationParameters":{"mode":"Overwrite","partitionBy":"[\"country\"]"},"isolationLevel":"Serializable","isBlindAppend":false,"operationMetrics":{"numFiles":"3","numOutputRows":"5","numOutputBytes":"3045"},"engineInfo":"Apache-Spark/3.4.0 Delta-Lake/2.4.0","txnId":"1cbc9537-63eb-4799-8647-2d947ae8fa41"}}
```

Comment:
taken from delta lake example, otherwise BQ API calls fail

https://github.com/delta-io/delta-examples/blob/master/data/people_countries_delta_dask/_delta_log/00000000000000000000.json